docs/reference/search/suggesters/completion-suggest.asciidoc

```diff
@@ -218,14 +222,15 @@ filtering but using suggest on the `_search` endpoint does:
 
 [source,js]
 --------------------------------------------------
-POST music/_search?size=0
```

Comment:
I see why this is confusing, but it's also an interesting piece of information that this `size` refers to the number of search hits to return. Maybe leave this and extend <3> to briefly explain the difference and why `size=0` makes sense here as the url parameter?

Reply:
Christoph and I talked about this and it seems like Elasticsearch is already optimized when suggest only search requests are executed (see `QueryPhase.execute(SearchContext searchContext)`), so it seems like adding the `size=0` does not really make sense.